VDA-Name-Registry/hardhat.config.ts

```diff
@@ -65,8 +65,8 @@ const config: HardhatUserConfig = {
       accounts: [PRIVATE_KEY!],
     },
     polygontestnet: {
-      // url: "http://44.234.36.28:8545",
-      url: "https://rpc-mumbai.maticvigil.com/",
+      url: "https://ca-admin:q9SBb1F58Lk275t0lPlKf4MN@polygon-mumbai.capturealpha.com",
```

Comment:
This needs to remain private and not committed to repo.

Reply:
Updated